libs/application/template-api-modules/src/lib/modules/templates/old-age-pension/old-age-pension.service.ts

```diff
@@ -9,24 +13,39 @@ import { BaseTemplateApiService } from '../../base-template-api.service'
 import { TemplateApiModuleActionProps } from '@island.is/application/template-api-modules'
 
 import {
-  HelloOddurApi,
   SocialInsuranceAdministrationClientService,
+  Uploads,
 } from '@island.is/clients/social-insurance-administration'
+import { transformApplicationToOldAgePensionDTO } from './old-age-pension-utils'
+import { getValueViaPath } from '@island.is/application/core'
+import { S3 } from 'aws-sdk'
+import { getApplicationAnswers } from '@island.is/application/templates/old-age-pension'
 
 interface customError {
```

Comment:
Breyta nafni í PascalCase til að fjarlægja eslint villu
```suggestion
interface CustomError {
```